failure to publish github release notes ' R: Publish GitHub Page'

https://github.com/checkstyle/checkstyle/runs/8179578319?check_suite_focus=true
```
Run ./.ci/publish-releasenotes-githubpage.sh 10.3.3
TARGET_VERSION=10.3.3
Error: Define  environment variable
Error: Process completed with exit code 1.
```

@romani Trying to replicate [the above comment's](https://github.com/checkstyle/checkstyle/issues/12169#issuecomment-1239374997) behavior on local but have had no success. Can you help with this?
```
./.ci/update-github-page.sh 10.3.3
```
```
Generation ends with 2 errors.

[ERROR] Issue #12351 does not have breaking compatibility, bug, miscellaneous, new feature, new module label! Please set label at https://github.com/checkstyle/checkstyle/issues/12351
[ERROR] Issue #12370 does not have breaking compatibility, bug, miscellaneous, new feature, new module label! Please set label at https://github.com/checkstyle/checkstyle/issues/12370

```

--- a/.ci/jsoref-spellchecker/whitelist.words
+++ b/.ci/jsoref-spellchecker/whitelist.words
@@ -1193,6 +1193,7 @@ rpmfind
 rq
 Rrridden
 RRRR
+Rsa
 rtc
 Rtl
 ru

--- a/.ci/update-github-page.sh
+++ b/.ci/update-github-page.sh
@@ -72,24 +72,24 @@ CONTENT=$(cat github_post.txt)
 echo "$CONTENT"
 
 echo 'Updating content to be be json value friendly'
-UPDATED_CONTENT=$(awk '{printf "%s\\n", $0}' github_post.txt |  sed "s/\`/'/g")
+UPDATED_CONTENT=$(awk '{printf "%s\n", $0}' github_post.txt | jq -Rsa .)
 echo "$UPDATED_CONTENT"
 
 RELEASE_ID=$(cat /var/tmp/cs-releases.json | jq -r ".[$TARGET_RELEASE_INDEX].id")
 echo RELEASE_ID="$RELEASE_ID"
 
-JSON=$(cat <<EOF
+cat <<EOF > body.json
 {
 "tag_name": "checkstyle-${TARGET_VERSION}",
-"body": "${UPDATED_CONTENT}"
+"body": ${UPDATED_CONTENT}
 }
 EOF
-)
-echo "$JSON"
+echo "JSON Body"
+cat body.json
 
 echo "Updating Github tag page"
 curl --fail-with-body \
   -X PATCH https://api.github.com/repos/checkstyle/checkstyle/releases/"$RELEASE_ID" \
   -H "Accept: application/vnd.github+json" \
   -H "Authorization: token $GITHUB_TOKEN" \
-  -d "${JSON}"
+  --data @body.json

